### DISCUSSION TITLE
How hard would it be to deploy to iOS

### DISCUSSION THREAD

**Discussion by abaybektursun** [RWKV/rwkv.cpp#56] [Q&A]
How hard would it be to get let's say `Q4_0` to run on an iOS device?

**Comment by saharNooby** [comment]
I guess as hard as it is to build anything C/C++ on iOS. Can't help with it though, I'm completely unfamiliar with Apple ecosystem.

**Comment by guinmoon** [comment]
> How hard would it be to get let's say `Q4_0` to run on an iOS device?

You can checkout [my project](https://github.com/guinmoon/LLMFarm). It's based on this great repo. You can replace the 20B tokenizer in the resources with your own if needed.
